src/test-reader/test-transformer.ts

```diff
@@ -0,0 +1,47 @@
+import * as babel from "@babel/core";
+import { addHook } from "pirates";
+
+import type { NodePath, PluginObj, TransformOptions } from "@babel/core";
+import type { ImportDeclaration } from "@babel/types";
+
+const TRANSFORM_EXTENSIONS = [".jsx", ".tsx"];
+
+export const setupTransformHook = (): VoidFunction => {
+    const transformOptions: TransformOptions = {
+        browserslistConfigFile: false,
+        babelrc: false,
+        configFile: false,
+        compact: false,
+        plugins: [
+            [
+                require("@babel/plugin-transform-react-jsx"),
+                {
+                    throwIfNamespace: false,
+                    runtime: "automatic",
+                },
+            ],
+            require("@babel/plugin-transform-modules-commonjs"),
+            [
+                (): PluginObj => ({
+                    name: "remove-css-imports",
+                    visitor: {
+                        ImportDeclaration(path: NodePath<ImportDeclaration>): void {
+                            if (path.node.source.value.match(/\.(css|less|scss)$/)) {
+                                path.remove();
```

Comment:
In order to remove css imports. I don't find solution how to transpile them, so I remove them because we need only read tests and not execute in nodejs (will be executed in browser in next PR)